### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by mdarif6**
completed Avatar component first

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  object-fit: cover;
+  /*   position:relative; */
+}
+
+.avatar-xsm {
+  width: 50px;
+  height: 50px;
```
Comment (anuj-kumary) [inline components/avatar.css]: Maintain consistency in units.

**Review by Aman-Jain1996 — COMMENTED** [review]
Arif please make index.html main page so that it can be viewed directly , else we need to write URL to check component , and use responsive units as much as possible

**Review by hrishi82 — COMMENTED** [review]
The code structure looks good to me

Suggestion: Try to use rems instead of px,

**Review by Hnh013 — APPROVED** [review]
looks fine ! a suggestion: give more spacing between components

**Review by lomesshh — APPROVED** [review]
Looks good !
you can merge it

**Pull request merged**